master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md

```diff
@@ -19,11 +19,11 @@ component needs access to in etcd to function successfully.
 | /calico/ipam/v2/\*                        |   RW   |
 | /calico/resources/v3/projectcalico.org/\* |   RW   |
 
-## felix as a stand alone process
+## Felix as a stand alone process
 
 | Path                                      | Access |
 |-------------------------------------------|--------|
-| /calico/felix/v1/\*                       |   RW   |
+| /calico/felix/v1/\*                       |   R    |
```

Comment:
I think this isn't quite right - `/calico/felix/v1/` is where Felix will write its status reports if enabled, so it does need that permission.

However, it doesn't need write access to `/calico/resources/v3`

Reply:
Oh OK, interesting. `/calico/resources/v3` is not mentioned in this table as being required, so I guess we do not need to make any change in that regard. 

Would you say then that this page is wrong https://docs.projectcalico.org/v2.6/reference/advanced/etcd-rbac/calico-etcdv2-paths and Felix actually needed `W` for `/calico/felix/v1/in v2.6?

Reply:
> Oh OK, interesting. /calico/resources/v3 is not mentioned in this table as being required, so I guess we do not need to make any change in that regard.

Pretty sure it is, on the next line.

Reply:
Oh dang, oops. LOL